Capstone/frontend-employee/src/Pages/ManagerDashboard/Employee/ManagerDisplayEmployee.jsx

```diff
@@ -0,0 +1,158 @@
+import React, {useState, useEffect} from 'react'
+import axios from "axios";
+import { Link, useNavigate } from "react-router-dom";
+import MultiSelectDropdown from '../../../component/MultiSelectDropdown/MultiSelectDropDown';
+import Skills from '../../../component/Data/skills';
+
+
+
+const ManagerDisplayEmployee = () => {
+    const [employees, setEmployees] = useState([]);
+    const [managerId_usestate, setmanagerId_usestate] = useState("");
+    const navigate = useNavigate();
+    const email = localStorage.getItem("email")
+    const [check,setCheck]=useState(false);
+    const [skills, setSkills] = useState([]);
+    const [selectedSkills, setSelectedSkills] = useState([]);
+    useEffect(() => {
+      getAllEmployees();
+    }, []);
+  
+    const getAllEmployees = async () => {
+      try {
+        const response = await axios.get(
+          "http://localhost:8080/user/all/EMPLOYEE"
+        );
+        console.log(response.data);
+        setEmployees(response.data);
+        response.data.forEach((employee)=> {
+          IsRequested(employee)
+        });
+        setmanagerId_usestate(employees.managerId);
+      } catch (error) {
+        console.error("Error fetching data:", error);
+      }
+    };
+    const getSkilledEmployee = async (skills, check) => {
+      try {
+        const response = await axios.get(
+          `http://localhost:8080/user/employees/skills?skills=${skills}&isCheck=${check}`
+        );
+        console.log(response.data);
+        setEmployees(response.data);
+      } catch (error) {
+        console.log(error);
+      }
+    };
+    const handleSkillChange = (selectedOptions) => {
+      const selectedSkillsValues = selectedOptions.map((option) => option.value);
+      setSkills(selectedSkillsValues);
+    };
+    const handleCheckChange = () => {
+      setCheck(!check);
+    };
+    const handleSkillClick = () => {
+      console.log(skills);
+      console.log(check);
+      getSkilledEmployee(skills, check);
+    };
+    const IsRequested = async (employeeObject) => {
+      try {
+        const response = await axios.post(
+          "http://localhost:8080/user/employee/isRequested",
```

Comment:
Use common functions for call actions for GET, POST, PUT methods. And don't use endpoints directly like this.